wgsl/index.bs

```diff
@@ -3955,19 +3984,9 @@ It will stabilize in a finite number of steps.
 
 ### Function attributes for entry points ### {#entry-point-attributes}
 
-: <dfn noexport>stage</dfn>
-:: The `stage` attribute declares that a function is an entry point for a particular pipeline stage.
-    The parameter must be one of `vertex`, `fragment`, or `compute`.
-: <dfn noexport>workgroup_size</dfn>
-:: The `workgroup_size` attribute specifies the x, y, and z dimensions of the [=workgroup grid=]
-    for a [=compute=] shader.
-    The size in the x dimension is provided by the first value.
-    The size in the y dimension is provided by the second value, when present, and otherwise
-    is assumed to be 1.
-    The size in the z dimension is provided by the third value, when present, and otherwise
-    is assumed to be 1.
-    Each dimension size must be at least 1 and at most an upper bound specified by the WebGPU API.
-    This attribute must only be used with a [=compute shader stage=] entry point.
+[SHORTNAME] defines the following attributes that can be applied to entry point declarations:
+ * [=stage=]
```

Comment:
Can we make it so the attributes are referred behind a namespace? i.e. `[=attributes/stag=]` instead of `[=stage=]`. Otherwise, seeing "stage" is overly generic here.